Library/Homebrew/api.rb

```diff
@@ -4,6 +4,7 @@
 require "api/analytics"
 require "api/cask"
 require "api/formula"
+require "development_tools"
```

Comment:
This particular one can probably be lazily loaded within `fetch_json_api_file` since that's the only place this is used.